### ISSUE TITLE
ProhibitUnusedImport doesn't recognize exported functions as being used

### ISSUE THREAD

**Issue by oalders** [gugod/Perl-Critic-TooMuchCode#18]
I may just do this myself, but I'm making a note here so I don't lose track of it.

```perl
use Mojo::JSON qw(decode_json encode_json);

  {
      our @EXPORT_OK = qw(
          decode_json
          encode_json
      );
  }
```

This will trigger the policy because it doesn't recognize those functions as having been used.

**Comment by gugod** [comment]
This is true only because `@EXPORT_OK` is a special variable though.

**Comment by gugod** [comment]
I just released 0.18 with the fix of this issue: https://gugod.org/2021/09/cpan-release-of-toomuchcode-0.18/
